Makefile

```diff
@@ -0,0 +1,64 @@
+LOCAL_IMAGE = acorn_docker:1.0
+REMOTE_IMAGE = merlinran/acorn_docker
+
+.PHONY: list
+list:
+	@echo "Welcome! You may want to try one of the following:\n---"; \
+	grep "^.PHONY:" Makefile | grep "#" | cut -d ":" -f 2- | sed "s/\w*#/\t#/g" | sed "s/^/make/"
+
+.PHONY: simulation # Run the vehicle and server containers in simulation mode.
+simulation: docker-image
+	@DOCKER_COMPOSE_FILE=docker-compose-simulation.yml make restart-docker-compose &&\
+	echo "Please visit http://localhost"
+
+.PHONY: attach-vehicle # Attach to the shell of the vehicle container. It creates the container if it doesn't exist.
+attach-vehicle:
+	@test -z `docker ps -f "name=acorn_vehicle" -q` && make docker-vehicle; \
+	docker exec -it acorn_vehicle /bin/sh
+
+.PHONY: attach-server # Attach to the shell of the server container. It creates the container if it doesn't exist.
+attach-server:
+	@test -z `docker ps -f "name=acorn_server" -q` && make docker-server; \
+	docker exec -it acorn_server /bin/sh
+
+.PHONY: stop # Stop both the vehicle and server containers.
+stop:
+	@docker-compose -f docker-compose-simulation.yml down --remove-orphans
+
+.PHONY: docker-test # Start the vehicle container in test mode and run the tests in it.
+docker-test: docker-image
+	@docker-compose -f docker-compose-test.yml up --remove-orphans -d && \
+	docker exec -it acorn_vehicle make test
+
+.PHONY: push-image # Build and push image to Docker Hub to be used by CI.
+push-image: docker-image
+ifeq ($(shell uname -m),arm64)
+	docker buildx build -t $(REMOTE_IMAGE) --platform linux/amd64,linux/arm64 --push .
```

Comment:
This is to allow me building and pushing the image for `linux/amd64` while on my MacBook with M1 processor. Otherwise it just hits the next branch.